src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -525,12 +525,17 @@ def _deploy_workflow(self, step_name: str, settings):
         return None
 
     def _upload_wheel(self):
+        wheel_paths = []
         with self._wheels:
-            return self._wheels.upload_to_wsfs()
+            if self._config.upload_dependencies:
+                wheel_paths = self._wheels.upload_wheel_dependencies(["databricks_sdk", "sqlglot"])
+            wheel_paths.append(f"/Workspace{self._wheels.upload_to_wsfs()}")
+            return wheel_paths
 
-    def _upload_wheel_runner(self, remote_wheel: str):
+    def _upload_wheel_runner(self, remote_wheel: list[str]):
         # TODO: we have to be doing this workaround until ES-897453 is solved in the platform
-        code = TEST_RUNNER_NOTEBOOK.format(remote_wheel=remote_wheel, config_file=self._config_file).encode("utf8")
+        remote_wheels_str = ", ".join(list(remote_wheel))
```

Comment:
Multiple pip install is with spaces, not with ", "

Reply:
@JCZuurmond Would our %pip cell also understand that, by the way? 

This is why we need an integration test here

Reply:
Currently not, but @ericvergnaud is working on [a change](https://github.com/databrickslabs/ucx/tree/support-pip-install-requirements) so that it will

Reply:
added space